src/styles/index.css

```diff
@@ -1,5 +1,7 @@
 body {
   margin: 0;
+  max-width: 80%;
```

Comment:
I'd refrain from setting styles on the body as much as possible. To keep things flexible, add a Layout component and import it wherever you want to apply this type of styling.